models/result_value.go

```diff
@@ -21,3 +21,13 @@ func (r *ResultValue) GetScalarType() string {
 		return "other"
 	}
 }
+
+type ResultValueWrap struct {
+	EntryWrap *Entry
+	Value     *ResultValue
+}
+
+func (e *Entry) WrapResultValue(val ResultValue) ResultValueWrap {
+	var rvWrap = ResultValueWrap{EntryWrap: e, Value: &val}
+	return rvWrap
```

Comment:
A couple things regarding this function and struct. If using a custom struct like this is how you want to do it, is there any reason you couldn't just make the struct contain an `Entry` and a `ResultValue` slice instead, and then go back to just having one `result_value` template that iterates over that slice?

Also, regardless, I think this method ought to go in `entry.go` since it's an `Entry` method